Landing hero/kevin

kevin200555 opened the pull request:
Pull request for the header changes

kevin200555 committed:
- Added a tempoary video file to the \Header folder to act as a placeholder
- Changed the margin styling for the "Cybersecruity matters..." Text so that
the video and the text fit on one screen
- uncommented back the description text
- I also changed the .lintstagedrc.json file.  This was because
video files would be sent to Prettier, which can't process those.
and this was stopping me from committing

kevin200555 committed:
- added a listItem to the home page, copied the styles
from the navbar items
- discord doesn't have its own icon in lucid-icons, so I replaced it with a messages one
- items still need functionality

kevin200555 committed:
- In Hackerlab.svelte, I gave one of the divs an id,
this is so that The Hackerlab button can scroll to it on click
- Added a color property to my ListItem, this way people
can easily tell if the lab is open or not
- the link to the discord now opens a new window
- other stlying implemented to make my altered
ListItem match the original in the NavBar

kevin200555 committed:
Added some support for smaller screens

kevin200555 committed:
- Decided to revert the .lintstagedrc.json file to how it was before, I think it would be better to merge before changing it.
- replaced the video with a placeholder image

kevin200555 committed:
- chnaged header to match the mockup more
- added functionality to events button

alexanderl19 reviewed (changes requested):
Generally your code looks good, but we should be targeting a more accurate version of the mockup.

It doesn't have to be pixel perfect, but at the minimum, the side margins should be reduced, and the buttons should fill the space responsively.

kevin200555 committed:
changed the colors of the list items, removed some
more whitespace (including from the navbar)

kevin200555 committed:
Updated the Landing Page with new styles
Added buttons for the social medias

kevin200555 committed:
changed hackerlab section and competition section
Also made it so that .lintstagedrc.json doesn't account
for images

kevin200555 committed:
changed hackerlab section and competition section
Also made it so that .lintstagedrc.json doesn't account
for images

kevin200555 committed:
Added events section

kevin200555 committed:
fix: Made website more responsive on mobile

kevin200555 committed:
fix:Added some changes that Akshay mentioned and also
I made the events part work

kevin200555 committed:
fix: changed the header to be more consistent with the rest of the page
added some changes for mobile

aanyal committed:
Updated Calendar (#394)

* Added basic calendar

* Fixed the calendar not showing up issue

* Configured the colors

* Added in events from Cyber Google Calendar

* Added side recent events

* Updated pnpm lock files

* Updated package.json

* Updated package.json to correct version of Schedule-X

* Cleaned up code, split it up to multiple file

* Configured different calendar types, and added colors

* Made font-sizes more consistent, changed colors

* Fixed pathing issue

* Fixed pathing issue

* Renamed the calendar api key

* Split up the code into multiple modules

* Reformatted side evet details

* Added scroll functionality

* Cleaned up the code

* Added colors and updated calendars

* Added calendar configuration to Sanity

* Added sanity code

* Added experience level

* Added event type on hove

* Added pnpm and package.son

kevin200555 committed:
feat: reformated the hackerlab section, also added a
hackerlab open/closed section

kevin200555 committed:
feat: added the homepage into sanity

kevin200555 committed:
Merge remote-tracking branch 'origin/main' into LandingHero/Kevin

kevin200555 committed:
feat: implement multi-calendar events section on homepage

Replaces single hardcoded ICS feed with per-calendar ICS fetching
driven by Sanity calendar documents. Upcoming events card now shows
the next 2 events with colored left borders matching their calendar
type. Header cover image is now sourced from Sanity.

aanyal committed:
Switch pics to Sanity, added animations and fixed formatting a little

aanyal committed:
Removed static images

aanyal committed:
Fixed pnpm-lock.yaml file

aanyal committed:
Edited responsiveness
